engines/wintermute/detection_tables.h

```diff
@@ -480,6 +480,64 @@ static const WMEGameDescription gameDescriptions[] = {
 	WME_WINENTRY("reversion1", "Version 1.3.2369",
 		WME_ENTRY2s("xlanguage_pt.dcp", "886886b6b14aadac844078de856799a6", 10620797,
 					"data.dcp", "aecb5deeea7b0baa871fbd0cef35a648", 254219204), Common::PT_BRA, ADGF_UNSTABLE, LATEST_VERSION),
+	
+	// Reversion: The Escape (Steam release) (Spanish)
+	//
+	// The data for the Spanish version are all in data.dcp.
+	// Because we rely on WME_ENTRY2s to detect the languages,
+	// we have to detect data.dcp twice. For some reason, the
+	// Spanish version of the game only works if **no** xlanguage_*.dcp
+	// file is present in the game directory. If any xlanguage_*.dcp
+	// file exists in the game directory, the detection entry for the
+	// Spanish version will launch another language instead.
+	//
+	// I (rootfather) think that the problem is that there's no
+	// special xlanguage_*.dcp file for Spanish. The original game
+	// seems to switch between languages by simply copying the desired
+	// language file from /languages to the main folder. 
+	//
+	// To enable Spanish speech, remove xvoice_en.dcp from your game directory.
+	// To enable English speech, copy xvoice_en.dcp from the /languages directory
+	// to the game directory.
+	WME_WINENTRY("reversion1", "Steam",
+		WME_ENTRY2s("data.dcp", "5e4d40075f69fa7702530e38c349d2fd", 254293949,
+					"data.dcp", "5e4d40075f69fa7702530e38c349d2fd", 254293949), Common::ES_ESP, ADGF_UNSTABLE, LATEST_VERSION),
```

Comment:
So, the thing here is that quite a few of the Wintermute-games are distributed with a bunch of language-files installed, and then they do language-selection by shuffling files prior to starting the engine, such that one language file is within reach of the engine.

Now, what you WANT to happen in that case, is that the detection detects ALL the language versions that are available, so that the user gets the options. (The engine itself then selects which files to ignore/load based on the language it gets passed from the launcher).

I assume that what's going on here, and the reason why I added the comment about not mixing WME_ENTRY1s and WME_ENTRY2s, is to ensure that you get all the languages detected no matter if you have 10 or 1 language file in the folder at the time when detection is run. (My assumption being that the WME_ENTRY1s hit will be ignored if you have 1 or more 2-entry hits).

So, the use case here is:
"User has installed game with i.e. xlanguage_en.dcp available in game folder, but would like to  play in Spanish too".

If my assumption is correct, then using the WME_ENTRY1s solution would lead to only detecting english, while doing the extra entry with WME_ENTRY2s would present the user with the option of choosing spanish as well.